app/assets/javascripts/detect_duplicates.js

```diff
@@ -10,7 +10,7 @@ function detect_duplicates() {
     $.ajax({ url: url + '&format=json' })
     .done(function(data) {
       if ( data.docs.length > 0 ) {
-        $me.parent().append('<div id="duplicates" class="alert alert-warning">This ID is already in use: <a href="' + url + '">view records using this Source Metadata ID</a>.</div>');
+        $me.parent().append('<div id="duplicates" class="alert alert-warning">This ID is already in use: <a href="' + url + '">view records using this Source Metadata ID</a>.  Please use a portion note to differentiate records with the same ID.</div>');
```

Comment:
Maybe "Please consider using the Portion Note field to help differentiate between objects with the same metadata" ?